packages/contract-wrappers/test/coordinator_wrapper_test.ts

```diff
@@ -138,15 +139,16 @@ describe.skip('CoordinatorWrapper', () => {
                 },
             },
             NETWORK_ID_TO_CONTRACT_ADDRESSES: {
-                [config.networkId]: contractAddresses,
+                // TODO: change to CHAIN_ID_TO_CONTRACT_ADDRESSES when @0x/coordinator-server is ready
```

Comment:
Did you open up an issue for this so we can track it?

Reply:
@hysz would be good to get this on your radar